mainmatter/introduction/graphics_hardware/index.tex

```diff
@@ -123,7 +123,45 @@ \subsubsection{NVIDIA}
 We were not able to find a whitepaper or similar report from reliable sources which detailed the GP106 chip, so that is why it is not used as the example. 
 
 
-\subsubsection{Intel}
+\subsubsection{AMD}
+For this project we have a AMD Sapphire Radeon R9 280 3GB GDDR5 \gls{GPU} at our disposal. It uses the GCN 1.0 Architecture.
+GCN stands for Graphics Core Next, it is the first architecture AMD developed that would allow to use \glspl{GPU} as \glspl{GPGPU}.
 
+\fig{figures/GCN_compute_unit}{Hello World}{GCNCU}{1} 
+
+In GCN Compute Units are the basic computational blocks of the GCN architecture, Figure \todo{ref fig} visualizes how each Compute Unit(CU) is structured in GCN.
+A wavefront consists of 64 threads that can be executed in parallel, these wavefronts are distributed amongst CUs. 
```

Comment:
Begrebet "Wavefront" bruges her, men defineres ikke rigtig, og det er heller ikke åbenlyst set fra figuren

Reply:
Don't know how to fix.